providers/common/ciphers/basic_ciphers.c

```diff
@@ -651,15 +652,88 @@ static int generic_ctr_cipher(PROV_GENERIC_KEY *dat, unsigned char *out,
 
 BLOCK_CIPHER_aes_generic_prov(cbc)
 BLOCK_CIPHER_aes_generic_prov(ecb)
-BLOCK_CIPHER_aes_generic_prov(ofb)
-BLOCK_CIPHER_aes_generic_prov(cfb)
+BLOCK_CIPHER_aes_generic_prov(ofb128)
+BLOCK_CIPHER_aes_generic_prov(cfb128)
 BLOCK_CIPHER_aes_generic_prov(cfb1)
 BLOCK_CIPHER_aes_generic_prov(cfb8)
 BLOCK_CIPHER_aes_generic_prov(ctr)
 
 #ifndef FIPS_MODE
 
 # ifndef OPENSSL_NO_ARIA
+
+/*
+ * TODO (3.0) The chunked code needs to move when other ciphers start using
+ * these methods. It is only here currently to avoid 'not used' errors.
+ */
```

Comment:
Wouldn't the chunked functions become good candidates for the `cipher_common.c` unit you made in #9580?  That means we should probably make sure to review and get that PR merged first...

Reply:
yes - correct

Reply:
ping : Updated PR quite a bit after a rebase